wolfssl/wolfcrypt/settings.h

```diff
@@ -202,7 +202,14 @@
 
 /* make sure old RNG name is used with CTaoCrypt FIPS */
 #ifdef HAVE_FIPS
-    #define WC_RNG RNG
+    #if defined(HAVE_FIPS_VERSION) && (HAVE_FIPS_VERSION < 2) || \
```

Comment:
This can be simplified down to:

    !defined(HAVE_VERSION) || (HAVE_FIPS_VERSION < 2)